app/router.js

```diff
@@ -6,6 +6,8 @@ const Router = EmberRouter.extend({
   rootURL: config.rootURL
 });
 
-Router.map(function() {});
+Router.map(function() {
+  this.route('about');
```

Comment:
I think we should keep the routes empty for default, I think is better add documentation in the readme about how work prember

example:
```js
const app = new EmberApp(defaults, {
  prember: {
    urls: [
      '/',
      '/about',
      '/contact'
    ]
  }
});
```

what do you think?

Reply:
yep, that's valid, I just didn't want them to miss it out, so maybe pointing it out in the README